compiler/rustc_middle/src/thir/visit.rs

```diff
@@ -1,6 +1,4 @@
-use rustc_middle::thir::{self, *};
-use rustc_middle::ty::Const;
-
+use super::*;
```

Comment:
how annoying is it to not use a glob import here? As we are in `rustc_middle` this glob import now also imports private use statements, which i like to avoid

Reply:
OH so tahts why i wasnt getting errors about `ty::Const` not existing